src/common/core/users/serialization.js

```diff
@@ -1177,10 +1174,16 @@ define(['common/util/assert', 'blob/BlobConfig'], function (ASSERT, BlobConfig)
                     }
                 },
                 updateSheet = function (name) {
-                    //the removed object should be already removed...
                     //if some element is extra in the place of import, then it stays untouched
                     var oldMemberGuids = Object.keys(oldSheets[name]),
-                        i;
+                        newMemberGuids = Object.keys(newSheets[name]);
+
+                    for (i = 0; i < newMemberGuids.length; i += 1) {
+                        if (oldMemberGuids.indexOf(newMemberGuids[i]) === -1 && nodes[newMemberGuids[i]]) {
+                            //it was only removed from the sheet
+                            core.delMember(root, name, newMemberGuids[i]);
+                        }
+                    }
 
```

Comment:
not entirely.
during guid checks it turned out, that we have a missing use-case in our portfolio